rotkehlchen/api/rest.py

```diff
@@ -3831,11 +3831,7 @@ def get_associated_locations(self) -> Response:
     def query_online_events(self, query_type: HistoryEventQueryType) -> dict[str, Any]:
         """Queries the specified event type for any new events and saves them in the DB"""
         try:
-            if query_type == HistoryEventQueryType.EXCHANGES:
-                self.rotkehlchen.exchange_manager.query_history_events()
-                return OK_RESULT
-
-            # else we query eth staking events
+            # query eth staking events
```

Comment:
As I asked here something does not make sense: https://discord.com/channels/657906918408585217/1080783409640648765/1315760941425496074

We need to decide how events for exchange deposit/withdrawals (and later other types) of stuff will be queried by the frontend.

Deleting this I am not sure if it's the right or wrong approach.

What you need to make sure is that there is a way for the frontend to refresh the deposit/withdrawals of each exchange (query latest) per exchange via an endpoint.

Not just history events, as that's too generic. Same we would have for trades etc.

I asked them for confirmation in discord as per the link

Reply:
I've added a new endpoint for querying history events for specific exchanges.

> Not just history events, as that's too generic. Same we would have for trades etc.

It would seem you're saying here that we need ways to query specific types of history events, but you mentioned in discord that 'having one generic per exchange is fine', so the new endpoint simply queries all history events from a specific exchange via the `query_history_events` function in the exchange interface.